src/main/kotlin/com/team4099/robot2020/subsystems/Feeder.kt

```diff
@@ -0,0 +1,101 @@
+package com.team4099.robot2020.subsystems
+
+import com.ctre.phoenix.motorcontrol.ControlMode
+import com.team4099.lib.logging.HelixEvents
+import com.team4099.lib.logging.HelixLogger
+import com.team4099.lib.motorcontroller.CTREMotorControllerFactory
+import com.team4099.lib.motorcontroller.SparkMaxControllerFactory
+import com.team4099.lib.subsystem.Subsystem
+import com.team4099.robot2020.config.Constants
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard
+
+object Feeder : Subsystem {
+    private val inMasterTalon = CTREMotorControllerFactory.createDefaultTalonSRX(Constants.Feeder.FEEDER_IN_MASTER_ID)
```

Comment:
two motor controllers should be SPARK MAX, the top one (what you call the "out" one) should be a Talon SRX